src/TLuaInterpreter.cpp

```diff
@@ -15558,7 +15558,7 @@ void TLuaInterpreter::initIndenterGlobals()
         if (lua_isstring(pIndenterState.get(), -1)) {
             e = tr("Lua error: %1.").arg(lua_tostring(pIndenterState.get(), -1));
         }
-        QString msg = tr("[ ERROR ] - Cannot load code formatter, indenting functionality won't be available.\n");
+        QString msg = qsl("%1\n").arg(tr("[ ERROR ] - Cannot load code formatter, indenting functionality won't be available."));
```

Comment:
This is a less than perfect message construction anyhow - the subsequence append could have been combined with this:
```diff
-    QString msg = qsl("%1\n").arg(tr("[ ERROR ] - Cannot load code formatter, indenting functionality won't be available."));
-    msg.append(e);
+    QString msg = qsl("%1\n%2").arg(tr("[ ERROR ] - Cannot load code formatter, indenting functionality won't be available.", e));
```

Reply:
> subsequence append could have been combined

Could have been earlier already, but wasn't. Feel free to upgrade later.